package/tests/testthat/_snaps/adal01/markdown-snaps.md

```diff
@@ -0,0 +1,21 @@
+# lsting
+
+```         
+Code
+  print(data_snap[[i]])
+Output
+  # A tibble: 24 × 9
+     USUBJID               VISIT      ISTPT   NFRLT RES_ADA  TU_ADA RES_NAB  TU_NAB AVAL_pk
+     <lstng_ky>            <lstng_ky> <fct>   <dbl> <chr>    <chr>  <chr>    <chr>  <chr>  
+   1 AB12345-RUS-16-id-165 Day 2      24H        24 Negative N/A    NA       N/A    0.00   
```

Comment:
@edelarua, do you know why AVAL_PK is 0.00? It should be some numbers or NA.

Reply:
@qiliu1013 I believe those `AVAL` values were set to 0 in `adpc` in `random.cdisc.data`. I have added a line in to convert 0 values as well as `NA`s to `"N/A"` for this column.